### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by zainsadaqat**
### In this pull request, I implemented the followings:

- `create-react-app`
- Setup Linters
- Use duck-pattern for folder structure
- Install `react-redux` and `react-router-dom`
- Create reusable functional components

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    </ul>
+    <form>
+      <input type="text" />
+      <input type="text" />
+      <input type="submit" value="Submit" />
+    </form>
+  </section>
+);
+
+export default Books;
```
Comment (EricMbouwe) [inline src/redux/books/books.js]: - [x] All components should be in the `components` folder and responsible for rendering data. The `redux` folder is used for component state management and therefore should not host the component itself. Please move this component to the component folder

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+const Categories = () => <h1>Under Construction</h1>;
+
+export default Categories;
```
Comment (EricMbouwe) [inline src/redux/categories/categories.js]: - [x] As mentioned above all components should be in the `components` folder and responsible for rendering data. The `redux` folder is used for component state management and therefore should not host the component itself. Please move this component to the component folder

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
### Hi STUDENT,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight!
- Good PR ✔️
- Good commits ✔️

# Required Changes ♻️ ⛔️
- [x] Add a descriptive readme, setup section, authors, acknowledgment, etc..., You can learn more about it and go from the template [here](https://github.com/microverseinc/readme-template)

- [x] You are missing two components in your project:
- `BookList` that renders a list of books using the Book component
- `CreateNewBook` that:
  * Renders a form with inputs for the title and the author of the book
  * Has a button to submit
    * the button wouldn't add the book *yet*, this is just a placeholder for now

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by henatan99 — APPROVED** [review]
Hi @zainsadaqat , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zainsadaqat** [comment]
### Thank you so much @henatan99 for approving this pull request 🍻 🍺

**Pull request merged**